lib/prelude/src/lib.rs

```diff
@@ -467,3 +467,31 @@ impl KnownTypeValue for False {
         false
     }
 }
+
+/// Alias for `for<'t> &'t Self : Into<T>`.
+pub trait RefInto<T> = where for<'t> &'t Self : Into<T>;
+
+
+
+// =============
+// === Owned ===
+// =============
+
+pub trait AsOwned {
```

Comment:
Missing documentation. And I was actually wondering what the "AsOwned" means when I saw its usages.

Reply:
Added!